pkg/support/materialize_test.go

```diff
@@ -0,0 +1,198 @@
+package support
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	ecv1beta1 "github.com/replicatedhq/embedded-cluster/kinds/apis/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/pkg/runtimeconfig"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMaterializeSupportBundleSpec(t *testing.T) {
+	// Create a test template that includes the conditional logic we want to test
+	testTemplate := `apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
```

Comment:
Any specific reason why we're using a test mock template VS using the actual `host-support-bundle.tmpl.yaml` file?